library/core/benches/ascii.rs

```diff
@@ -66,6 +66,8 @@ macro_rules! benches {
 use test::black_box;
 use test::Bencher;
 
+const ASCII_CASE_MASK: u8 = 0b0010_0000;
```

Comment:
Can't we just import this?

Reply:
We could if we're prepared to make it public but we'd need to mark it as unstable I would assume. Do we want to make this const public for all to use?